custom/ilsgateway/api.py

```diff
@@ -260,6 +267,8 @@ def location_sync(self, ilsgateway_location, fetch_groups=False):
             historical_groups = {}
             while counter != 5:
                 try:
+                    # todo: we may be able to avoid this call by passing the groups in as part of the original
```

Comment:
@czue I think that fetching groups in chunk that contains 100 locations, always makes timeout.